doc/examples/perception_pipeline/perception_pipeline_tutorial.rst

```diff
@@ -99,23 +112,24 @@ Save this file in the config folder in the robot's moveit_config package with na
 
 * *shadow_threshold*: The minimum brightness of the shadow map below an entity for its dynamic shadow to be visible
 
-* *padding_offset*: The size of the padding (in cm).
+* *padding_scale*: Should always be >= 1.0. Scale up collision shapes in the scene before excluding them from the octomap.
 
-* *padding_scale*: The scale of the padding.
+* *padding_offset*: Absolute padding (in m) around scaled collision shapes when excluding them from the octomap.
 
 * *filtered_cloud_topic*: The topic on which the filtered cloud will be published (mainly for debugging). The filtering cloud is the resultant cloud after self-filtering has been performed.
 
+* *ns*: An optional namespace for the advertised topics. Required for multiple sensors of the same type.
 
 Update the launch file
 ++++++++++++++++++++++
 
 Add the YAML file to the launch script
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-You will now need to update the *sensor_manager.launch* file in the "launch" directory of your panda_moveit_config directory with this sensor information (this file is auto-generated by the Setup Assistant but is empty). You will need to add the following line into that file to configure the set of sensor sources for MoveIt to use: ::
+You will now need to create a *sensor_manager.launch* file in the "launch" directory of your panda_moveit_config directory (e.g. `on github <https://github.com/ros-planning/panda_moveit_config/blob/rolling-devel/launch/sensor_manager.launch.xml>`_) with this sensor information. You will need to add the following line into that file to configure the set of sensor sources for MoveIt to use: ::
```

Comment:
This section is a bit ambiguous.
* The name of the file should be `sensor_manager.launch.xml` (.xml is missing)
* You reference the `panda_moveit_config` from ros-planning, but in general I believe this is not used. For the tutorials, this package from https://github.com/ros-planning/moveit_resources/tree/master/panda_moveit_config is used instead, and for the real robot, Franka's own package is used: https://github.com/frankaemika/franka_ros2/tree/humble/franka_moveit_config.
* Your fork of the aforementioned repo does not yet contain this launch XML. This could of course be because the reader is expected to create it themselves, but I believe it makes for a better tutorial to show something that already exists and works.

Reply:
TLDR I'll address the issues here :+1: 

> - You reference the panda_moveit_config from ros-planning, but in general I believe this is not used. For the tutorials, this package from https://github.com/ros-planning/moveit_resources/tree/master/panda_moveit_config is used instead, and for the real robot, Franka's own package is used: https://github.com/frankaemika/franka_ros2/tree/humble/franka_moveit_config.

Reasonbale concern. At the tiem of writing there was unclarity, but now we have it clarified https://github.com/ros-planning/moveit2_tutorials/issues/704 so I'll have to update this PR to use panda config from `moveit2_resources`, not from `panda_moveit_config` repo.